src/java.base/share/native/libverify/check_code.c

```diff
@@ -468,7 +468,8 @@ static void CCout_of_memory (context_type *);
 /* Because we can longjmp any time, we need to be very careful about
  * remembering what needs to be freed. */
 
-static void check_and_push(context_type *context, const void *ptr, int kind);
+static void check_and_push_malloc_block(context_type *context, void *ptr);
+static void check_and_push_string_utf(context_type *context, const void *ptr);
```

Comment:
Can't this be:

`static void check_and_push_string_utf(context_type *context, const char* str);`

Reply:
Indeed it can. Fixed.